### ISSUE TITLE
Exception on WebView2 with WinUI 3 Preview 4

### ISSUE THREAD

**Issue by mvegaca** [microsoft/microsoft-ui-xaml#4206]
Hi, I'm Martin from Windows Template Studio Team.

We've updated WinUI Templates to Preview 4 and we are having an issue with the WevView2 control that we didn't have on the preview 3.

**Describe the bug**

WebView2 control throws an exception while debugging. The app does not crash and if you press F5 on the exception details view, the app continues running and the webview content is shown.

**Steps to reproduce the bug**
1. Run the app with F5 (debugging)
2. See the app crash
3. Press F5 on the exception details in VS and keep using the app.

**Screenshots**
![image](https://user-images.githubusercontent.com/5556784/108058939-08606c00-7055-11eb-82cd-8851172759e4.png)

![image](https://user-images.githubusercontent.com/5556784/108059001-21691d00-7055-11eb-9f4e-0025089fe7bb.png)

**Version Info**

NuGet package version: 
[Microsoft.WinUI 3.0.0-preview4.210210.4] If you seeing your issue in older previews, please try preview4.

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
| - | Yes |

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| October 2020 Update (19042)        | Yes   |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |

**Comment by mvegaca** [comment]
I forgot to attach the minimum repro app.

[WebViewIssueWinUIApp.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5988331/WebViewIssueWinUIApp.zip)

**Comment by shelllet** [comment]
winui3-preview3 is ok.
but winui3-preview4: webview2 will throw Exception: WinRT originate error - 0x80070490 : 'Element not found.'.

**Comment by shelllet** [comment]
if i unistall webview2 runtime  and keep  microsoft edge brower throw exception :WinRT originate error - 0x80070002 : 'The system cannot find the file specified.'.

**Comment by boostafazoo** [comment]
```
Microsoft.UI.Xaml.Controls.dll!winrt::throw_hresult(struct winrt::hresult)
Microsoft.UI.Xaml.Controls.dll!winrt::impl::consume_Windows_UI_ViewManagement_IAccessibilitySettings<struct winrt::Windows::UI::ViewManagement::IAccessibilitySettings>::HighContrastChanged(struct winrt::auto_revoke_t,struct winrt::Windows::Foundation::TypedEventHandler<struct winrt::Windows::UI::ViewManagement::AccessibilitySettings,struct winrt::Windows::Foundation::IInspectable> const &)
Microsoft.UI.Xaml.Controls.dll!WebView2::TryCompleteInitialization(void)  
```

According to the [Known Issues for Preview 4](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/), the AccessibilitySettings.HighContrastChanged event is no longer supported in Desktop apps.

**Comment by AndreMarschalek** [comment]
just to mention: (same environment as @mvegaca)
```
Exception thrown at 0x00007FFA6C11D759 (KernelBase.dll) in Downloader.exe: WinRT originate error - 0x80070490 : 'Element not found.'.
onecore\com\combase\winrt\error\restrictederror.cpp(1016)\combase.dll!00007FFA6D0BC8D3: (caller: 00007FFA6D0C65A9) ReturnHr(4) tid(280c) 8007007E The specified module could not be found.
Exception thrown at 0x00007FFA6C11D759 in Downloader.exe: Microsoft C++ exception: winrt::hresult_error at memory location 0x000000CFBFD7C690.
```

**Comment by sibille** [comment]
This issue is solved after updating to Project Reunion 0.5 Preview

**Comment by codendone** [comment]
The fix is indeed in the Project Reunion 0.5 Preview. Thanks for confirming, @sibille!
